.circleci/config.yml

```diff
@@ -21,9 +21,13 @@ jobs:
     steps:
       - checkout:
           path: ~/superchain-registry
-      - go/load-cache
+      - restore_cache:
+          key: v1-{{checksum "go.work"}}
       - go/mod-download
-      - go/save-cache
+      - save_cache:
+          key: v1-{{checksum "go.work"}}
```

Comment:
Actually I don't think `go.work` is a good choice as a cache key. I had probably meant to use `go.work.sum`, but I don't believe that is a good choice either.

No matter for now, as the short term plan is to rip out caching.